client/src/components/header.tsx

```diff
@@ -73,17 +74,22 @@ export default function HeaderNav() {
         </Text>
 
         <QNavLink icon={<IconHome />} link="/home" label="Home" />
-        <QNavLink
-          icon={<IconDeviceDesktopQuestion />}
-          link="/ticket"
-          label="Ticket"
-        />
+        {role !== "mentor" && (
+          <QNavLink
+            icon={<IconDeviceDesktopQuestion />}
+            link="/ticket"
+            label="Ticket"
+          />
+        )}
```

Comment:
this is a pickier comment but the formatting here is slightly different from the other QNavLinks
I think if you format the document it should look a bit more consistent
(open command palette -> "format document")

Reply:
(feel free to also do this in other files)